Fixes and improvements

alkiko opened the pull request:
## Changes
Need:
- [x] Scrolle ned skal ikke være røde
- [ ] tekst links  
- [X] Call to action på cover video - Afspil film / går til link - ingen links i test
- [X] Vælg mellem en af de 5 brandfaver når man laver en knap
- [X] Billeder ved siden af hinanden
- [X] Flere typer knapper. Med fyld, uden fyld, farver
- [ ] Små billeder / ikoner ---- undersøg om fullwidth billedet skal ikke altid fylde det hele - eller kan vi lave max størrelse
- [X] Flere knapper på en post - lav knap component som kan tilføj X antal knapper
- [X] Max width på container / Content kort 
- [X] Headet tekst må ikke ligge på logo
- [X] Skift farve på content box
- [X] Størrelse på content box
- [ ] pile ved feature list i center ? 
- [X] man skal kunne se overskrift på Page list- vises overhovedet ikke - den lille grå pil  
- [X] Tænd eller sluk fade på cover video / foto - billedet vises med fade - video vises med fade - boolean tillad knap
- [ ] Tekster på top video - anders skriver manuel break  

Nice:
- [ ] Mellemrum mellem billeder på feature list mindre
- [ ] Mikkel`s element

ddennis committed:
chore: added prettier module

ddennis committed:
chore: gitignore webstore config files

ddennis committed:
chore: use dotenv to handle .env files

ddennis committed:
chore: added comment about using windows for development

ddennis committed:
feat: add arrowColor to CMS with default value

ddennis committed:
feat: implemented arrowColor on frontpage

ddennis committed:
feat: implemented arrowColor on page template

ddennis committed:
chore: yarn.lock updated

ddennis committed:
feat: color picker on textImage, full-width-image and  text-video

ddennis committed:
fix: remove durationFadeIn from gatsby image. Was throwing errors

ddennis committed:
feat: implement text color and background color on Buttun and external link

ddennis committed:
chore: add ability to add default color array as in env file

ddennis committed:
refactor: default colors on content background

ddennis committed:
chore: todo in readme

ddennis committed:
feat: add ImageImage component

ddennis committed:
refactor: add multiple btns component in cms and typedef

ddennis committed:
feat: add multiple buttons component

ddennis committed:
refactor: implement multiple btns in cms on components

ddennis committed:
refactor: implement buttonList on components

ddennis committed:
chore: added content

ddennis committed:
chore: update typedef to accept buttons

ddennis committed:
refactor: handle multiple buttons in header

ddennis committed:
chore: cleanup

ddennis committed:
refactor: handle btn outline

ddennis committed:
refactor: maxWidth on footer

ddennis committed:
feat: add section title to pageList

ddennis committed:
refactor: implement buttonlist on generic page

anders-sondergaard commented:
Når jeg prøver at lave en image / image ting med 2 knapper for jeg følgende fejl når jeg prøver at se siden
![image-1](https://user-images.githubusercontent.com/75014940/133094234-0c0d305c-6325-4f8a-a157-863d0cca7943.png)
![image](https://user-images.githubusercontent.com/75014940/133094239-e354e3ad-e5c0-470d-8799-39062a91b5c6.png)

anders-sondergaard commented:
@ddennis Har du set denne?

ddennis commented:
@anders-sondergaard  yes. Kigger på den.

anders-sondergaard reviewed (changes requested):
Noget dato på dette?

ddennis commented:
![image](https://user-images.githubusercontent.com/300269/134164952-a19a1ecb-6969-4bf4-839d-520ea2547f7f.png)

Kan du beskrive den bedre. I mit eksemple virker det fint.

ddennis commented:
Jeg fandt den. Fejlen sker på forsiden - jeg fixer

anders-sondergaard commented:
coolilo

On Tue, 21 Sept 2021 at 13:50, ddennis - Dennis Christoffersen <
***@***.***> wrote:

> Jeg fandt den. Fejlen sker på forsiden - jeg fixer
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cinemataztic/gatsby-theme-cinemataztic-cms/pull/15#issuecomment-923906133>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR4KGHBDFFTMNXUMTTSXS4TUDBWSHANCNFSM5D5X5AAA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Sincerely


*Anders Søndergaard*

   - +45 53587129
   - ***@***.***
   - Skype: sondergaard.anders

CinemaTaztic Aps

Skudehavnsvej 27B
2150 Nordhavn - Denmark

CVR: 37207187


Find us on: FaceBook <https://www.facebook.com/CinemaTaztic> - Instagram
<https://www.instagram.com/cinemataztic/> - Twitter
<https://twitter.com/cinemataztic>
More on: www.cinemataztic.com

-- 










Privileged/Confidential Information may be contained in this 
message. If you are not the addressee indicated in this message (or 
responsible for delivery of the message to such person), you may not copy 
or deliver this message to anyone. In such case, you should destroy this 
message and kindly notify the sender by reply email. Please advise 
immediately if you or your employer does not consent to email for messages 
of this kind. Opinions, conclusions and other information in this message 
that do not relate to the official business of CinemaTaztic and/or other 
members of the CinemaTaztic group of companies shall be understood as 
neither given nor endorsed by it. 

ddennis committed:
fix: missing query for imageImage component on frontpage

ddennis committed:
chore: add note about deprecated component

ddennis committed:
chore: example content updated

ddennis commented:
Den er fixet 😄

Commit:
Add ImageImage component to Example Child Page.

alkiko commented:
Jeg får fejl på http://localhost:8000/example-page/example-child-page nu @ddennis. 

Det ligner den tidligere fejl, men synes det ligner du har tilføjet firstImage og secondImage til query på page.js. 

Kan du gennemskue det?

<img width="1274" alt="Screenshot 2021-09-22 at 14 58 58" src="https://user-images.githubusercontent.com/6972570/134347896-f7f85599-de6a-4f5b-8a59-93f0f1afaa8d.png">
<img width="1277" alt="Screenshot 2021-09-22 at 14 59 17" src="https://user-images.githubusercontent.com/6972570/134347988-421ba64d-ea3c-41d3-b498-3d375ddee6dd.png">
<img width="1278" alt="Screenshot 2021-09-22 at 15 03 11" src="https://user-images.githubusercontent.com/6972570/134348525-e5955431-c5af-4a0b-83c4-e8dcecc658c8.png">

ddennis committed:
fix: validate if images exist in imageImage component

ddennis commented:
It has been fixed. I simply forgot to verify if the component also had images.

Commit:
Add develop:lan to example workspace  package.josn
This enables testing on other machines on same network.

alkiko commented:
![Screenshot from 2021-09-28 16-49-59](https://user-images.githubusercontent.com/6972570/135111360-69f218fd-5d84-4ef2-8969-dcbf4a5f06e0.png)
![Screenshot from 2021-09-28 16-50-21](https://user-images.githubusercontent.com/6972570/135111364-d9ec47ff-e010-40b2-9745-263236b8573b.png)
@ddennis får `Error: item has illegal 'type' property: 'ImageImage'` - både når jeg kigger frontpage og en alm page i netlifyCMS.

ddennis commented:
Nej. Jeg har ingen fejl hos mig.

Commit:
Update ImageImage example component position and text.

Commit:
0.11.9-rc.0

Commit:
Bump to prerelease rc.0

Commit:
Remove github npm registry from build.

Commit:
Upgrade gatsby-plugin-netlify-cms to 5.14.0
due to error: ObjectMiddleware.register: serializer for mini-css-extract-plugin/dist/CssModule/null is already registered
https://github.com/gatsbyjs/gatsby/issues/31102

Commit:
Version bump.

Commit:
Update frontpage header.

Commit:
Upgrade gatsbyjs version (upgrading gatsby-plugin-netlify-cms is apparently not enough).

Commit:
version bump.

Commit:
Set manualInit=true in gatsby-plugin-netlify-cms

Commit:
Version bump.

Commit:
process.env.GATSBY_BRAND_COLORS optional
otherwise CMS initialization fails when not set.

Commit:
version bump

alkiko commented:
Har lagt det nye tema live på https://www.cinemataztic.com/admin/ så du kan teste @anders-sondergaard.

Commit:
Change /example-page down-arrow color to green.

ddennis committed:
chore: create video page for testing

ddennis committed:
fix: empty button was shown - when text was missing

alkiko committed:
0.11.9-rc.1

alkiko committed:
0.11.9-alpha.0

alkiko committed:
Version bump.

alkiko committed:
Disable URL picker from file/image/video widget in CMS.

alkiko committed:
Version bump.

alkiko commented:
Har bygget og lagt den nye theme version på www.cinemataztic.com. 

@anders-sondergaard det er din video på [wheeloffortune-siden](https://github.com/cinemataztic/www.cinemataztic.com/blob/0e9424e1e43b7926a809683f2711c1cc1d1f4c59/site/content/pages/wheeloffortune.yml#L35) der får build til at crashe. Sitet understøtter ikke at denne video hentes fra en url - det skal være en video der uploades direkte i CMS'et. 

Ændringer: 
- Jeg har opdateret CMS så det ikke længere tillader URLs der hvor den bagvedliggende kode ikke understøtter det. Muligheden for at vælge en URL er noget CMS'et utilsigtet har tilføjet i en opdatering.
- Jeg har fjernet wheeloffortune-siden som bare så ud til at være en kladde med en masse lorem ipsum og standardværdier. 

Det burde være muligt at teste @ddennis' ændringer nu.

alkiko commented:
Skal vi afslutte denne opgave @anders-sondergaard? Det er den version du ser på www.cinemataztic.com nu og har brugt siden 3. januar. 

Eller har du stadig et udestående med @ddennis i forhold til listen? 
https://github.com/cinemataztic/gatsby-theme-cinemataztic-cms/pull/15#issue-994851863

alkiko commented:
@anders-sondergaard?

alkiko commented:
Closed without merging due to inactivity.
